lib/google/ads/google_ads/v0/services/recommendation_service_client.rb

```diff
@@ -286,11 +286,11 @@ def apply_recommendation \
             #   operations will return errors. If false, operations will be carried in a
             #   single transaction if and only if they are all valid.
             #   Default is false.
-            # @param operations [Array<Google::Ads::GoogleAds::V0::Services::DismissRecommendationRequest::DismissRecommendationOperation | Hash>]
+            # @param operations [Array<Google::Ads::GoogleAds::V0::Services::DismissRecommendationOperation | Hash>]
```

Comment:
Please don't change anything in the v0/ folder. These are all automatically-generated.

Reply:
I don't think it can be fixed without those changes in v0 folder but thanks for the quick response.